Assets/Scripts/GoalBoxController.cs

```diff
@@ -75,6 +76,7 @@ void Update()
                 gateAnimator.Play("CloseGateDoors");
                 goalSphere.SetActive(false);
                 sheepRemainingText.text = "";
+                GameObject.Find("/BasicGame/" + gameObject.name + "/closedGateNav").SetActive(true);
```

Comment:
I don't love this, it seems very fragile in the sense that it relies on the names of `BasicGame` and `closedGateNav` never changing. It should be easy to have an editor reference to `closedGateNav` instead since it's inside this same prefab and will not have to be reassigned between scenes.